dashboard/test/ui/runner.rb

```diff
@@ -313,7 +320,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if $options.log_dev
+      HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    end
```

Comment:
Why do we need this new log_dev option? You can already toggle this via CDO.hip_chat_logging (which is default off for developers).